libraries/AP_Generator/AP_Generator_IE_2400.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "AP_Generator_IE_FuelCell.h"
+
+#ifndef GENERATOR_ENABLED
+#define GENERATOR_ENABLED !HAL_MINIMIZE_FEATURES
+#endif
+
+#if GENERATOR_ENABLED
+
+class AP_Generator_IE_2400 : public AP_Generator_IE_FuelCell
+{
+    // Inherit constructor
+    using AP_Generator_IE_FuelCell::AP_Generator_IE_FuelCell;
+
+public:
+    void update(void) override;
+
+
+private:
+    // Process characters recieved and build sentence for IE 2.4kW
```

Comment:
Is this really what it does?  "build sentence"s?

Perhaps "extract terms" might be better?